static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -397,12 +397,15 @@ $gutter-large: 55px;
     // Unset ul from user agent stylesheet
     margin-bottom: 0;
     text-transform: lowercase;
+    display: flex;
```

Comment:
I also should put a comment here to explain this is for the reordering which is added in JS.